### PR TITLE
add SOP for new ontology requests

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
and try to add it to the navigation bar

close https://github.com/OBOFoundry/OBOFoundry.github.io/issues/1588

I'm not sure if I did this correctly - adding this folder to the nav bar.

Thanks for reviewing!

### CODE REVIEW

File: docs/SOP/NewOntologyRequest.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
```
Comment (matentzn) [inline docs/SOP/NewOntologyRequest.md:1]: This document should probably be simply `docs/SOP.md` and the New Ontology Request handling should be a section in it?

Comment (nicolevasilevsky) [reply docs/SOP/NewOntologyRequest.md:1]: ah, sure, i can do that. I'll fix it now

**Review by matentzn — APPROVED** [review]
Great, thank you! I would like @nataled to review the wording as well..

File: docs/SOP/NewOntologyRequest.md (hunk 1 above)
Comment (nicolevasilevsky) [reply docs/SOP/NewOntologyRequest.md]: done

**Review by nataled — COMMENTED** [review]
My comments:
1) I'd like to see an introduction to the SOP page, indicating that these are 'internal' instructions. At least it seems that way.

2) Remove the unnecessary 'issue request' after '(NOR)' in the first sentence.

3) Who thanks the submitter? Will it be what is later called the dashboard curator?

4) Is 'curator' the appropriate word for the person that adds the submission to the NOR dashboard? Perhaps 'administrator'?

5) "a reviewer will be assigned" - I'd prefer that 'reviewer' not be used here, as it has a very different meaning internally. Perhaps, given the duties, 'liaison' would be more appropriate. BTW, would be good to have a draft letter for the purpose of initial followup that could be linked from here. 

6) Replace 'NOR shepherd' with 'liaison' elsewhere as well.

7) Replace 'is volunteered' with 'is selected'

8) Replace 'This shepherd familiarizes herself with the new ontology request' with 'This liaison becomes familiar with the new ontology'

9) Replace 'answered question' with 'answers questions'

10) Remove the duplicated wording (everything after 'clarification and fixes from the submitter.'

**Comment by nicolevasilevsky** [comment]
> 3\. Who thanks the submitter? Will it be what is later called the dashboard curator?

I think the submitter is the ontology owner who submits the ontology request.

**Comment by nicolevasilevsky** [comment]
I made the @nataled's suggested changes. @matentzn this is ready for review again.

**Comment by nataled** [comment]
@nicolevasilevsky I referred to the person from the Foundry that thanks the submitter.

**Comment by nicolevasilevsky** [comment]
@nataled I misread that, thanks for clarifying!

**Comment by nicolevasilevsky** [comment]
I revised this to: 1. When receiving a new ontology request (NOR), the OBO dashboard administrator should thank the submitter for their submission.

**Review by nataled — APPROVED** [review]

**Pull request merged**